include/fakeclock/fakeclock.h

```diff
@@ -18,10 +18,18 @@ class FakeClock
     static time_point now() noexcept;
 };
 
+enum class ClockType
+{
+    MONOTONIC,
+    NON_MONOTONIC
+};
 class MasterOfTime
 {
+
   public:
-    MasterOfTime();
+    /// Master of time is monotonic when by creating it, you are not going back in time.
+    /// Non monotonic master of time will set current time to 1 second after epoch.
+    MasterOfTime(ClockType type = ClockType::NON_MONOTONIC);
```

Comment:
It's a bit confusing. The clock will be monotonic, but there is a discontinuity at the time of creation of `MasterOfTime`. I'd call it "StartTime" or "InitialTime" or something. And it could take "RANDOM" or "MONOTONIC" for now.